### PR TITLE
fix(notifier): adds provider validation based on hostname

### PR DESCRIPTION
**Pull request by vjgene**
https://rsklabs.atlassian.net/jira/software/projects/RMKT/boards/60?selectedIssue=RMKT-864

### CODE REVIEW

**Review by 0xartem — APPROVED** [review]
lgtm

File: src/services/notifier/hooks/providers.hook.ts [hunk 1]
```diff
@@ -33,9 +34,27 @@ export default {
...
+          ],
+          where: {}
+        }
+        return context
+      },
+      (context: HookContext) => {
+        if (!context.params.query) return context
+        const paramsSeq = context.params.sequelize
+        const sequelize: Sequelize = context.app.get('sequelize')
+        const { ...query } = context.params.query
```
Comment (jurajpiar) [inline src/services/notifier/hooks/providers.hook.ts]:
```suggestion
        const query = context.params.query
```

**Review by jurajpiar — APPROVED** [review]

File: src/services/notifier/hooks/providers.hook.ts (hunk 1 above)
Comment (itofarina) [inline src/services/notifier/hooks/providers.hook.ts]: Or:
```suggestion
        const { query } = context.params
```

**Review by itofarina — COMMENTED** [review]
Looks good!

File: src/services/notifier/hooks/providers.hook.ts [hunk 2]
```diff
@@ -33,9 +34,27 @@ export default {
                 }
               ]
             }
+          ],
+          where: {}
+        }
+        return context
+      },
+      (context: HookContext) => {
+        if (!context.params.query) return context
```
Comment (jurajpiar) [inline src/services/notifier/hooks/providers.hook.ts:43]: I think the last function should not return anything.

**Review by jurajpiar — CHANGES_REQUESTED** [review]

**Review by jurajpiar — APPROVED** [review]

**Pull request merged**